spec/cucumber/formatter/json_spec.rb

```diff
@@ -550,8 +550,8 @@ module Formatter
             FEATURE
 
           define_steps do
-            Given(/^there are bananas$/) { data = 'YWJj'
-              embed data, 'mime-type;base64' }
+            Given(/^there are bananas$/) { embed('YWJj', 'mime-type;base64') }
+
```

Comment:
Rogue line

Reply:
@luke-hill This line has to be here otherwise random tests _after_ this one fail. Seems to be a really fragile file. :( We can work on fixing that in another issue.